vinhloc30796/creativecontact-app
# refactor!: replace external SVG import with inline SVG in CreativeCon…

## Description

Converts the CreativeContactLogo component from using an external SVG file via @svgr/webpack to an inline SVG implementation. This change provides better control over the SVG rendering and removes the external dependency on the SVG file.

## Key Changes

1. Replaced external SVG import with inline SVG implementation
2. Simplified component props interface to only include essential properties
3. Moved @svgr/webpack from dependencies to devDependencies
4. Updated webpack configuration for better SVG handling

## Breaking Changes

- Changed CreativeContactLogoProps interface to only include `className`, `width`, and `height`
- Removed support for passing through arbitrary SVG props

## Related Issues

- Closes WEB-250

## Testing Instructions

1. Import and render the CreativeContactLogo component
2. Verify the logo renders correctly at different sizes
3. Confirm the logo maintains proper aspect ratio
4. Test className prop for custom styling

## Additional Notes

- The inline SVG implementation provides better control over the logo rendering
- This change reduces the build complexity by removing the need for external SVG processing
- Performance impact should be minimal as the SVG code is now part of the JavaScript bundle
